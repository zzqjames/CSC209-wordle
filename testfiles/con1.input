g i 1 
g h 4
y 2 -gy-g -g-gg birth
c br
c stx