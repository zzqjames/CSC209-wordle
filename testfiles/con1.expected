cannot_be: b r s t x 
must_be
[0] 
[1] i 
[2] t 
[3] 
[4] h 
